src/app/app.component.scss

```diff
@@ -3,6 +3,10 @@
   flex-direction: column;
 }
 
+a {
+  color: white;
```

Comment:
Bitte keine Farbkeywords verwenden. Teilweise interpretieren die Browser dies unterschiedlich oder können nicht alle. (https://stackoverflow.com/questions/3230851/are-there-any-cons-to-using-color-names-in-place-of-color-codes-in-css)
Am besten die Material Theming verwenden: https://material.angular.io/guide/theming